src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -255,3 +255,22 @@ export const sortStatementsByDate = statements => {
       format(new Date(a.pSStatementDateOutput), dateFormat),
   );
 };
+
+export const getLatestCopay = statements => {
+  return statements
+    ? statements.reduce((acc, currentCopay) => {
+        if (currentCopay.pSStatementDateOutput) {
+          if (!acc) {
+            return currentCopay;
+          }
+          return isAfter(
+            new Date(acc.pSStatementDateOutput),
+            new Date(currentCopay.pSStatementDateOutput),
+          )
+            ? acc
+            : currentCopay;
+        }
+        return acc;
+      }, null)
+    : null;
+};
```

Comment:
TODO: Will probably be good to add a unit test around this functionality.
Non-blocking, but when we look at upping test coverage for AuthdExp, we should make sure to cover this.

Reply:
will check in w @ajakabcin today to see if we have an epic for this already!